Package.swift

```diff
@@ -48,9 +76,14 @@ let package = Package(
             name: "GliaWidgetsSDK",
             dependencies: [
                 "SalemoveSDK",
-                "GliaWidgets",
+                "ReactiveSwift",
```

Comment:
I think that we should explicitly mention this in our docs that integrators have the possibility to use these dependencies. As we rarely update these dependencies versions so maybe even the version would be useful and it shouldn't be a pain to maintain. Also, do they need to do something to use their own version of the same dependency without any issues? Both previous versions and more recent versions.